src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -22,11 +22,45 @@ class FacilityTypeDropdown extends Component {
     this.state = {
       showBubble: false,
     };
+
+    // eslint-disable-next-line prettier/prettier
+    this.setCCInfoRef = (element) => {
+      this.ccInfoIcon = element;
+    };
+    this.focusCCInfoIcon = () => {
+      if (this.ccInfoIcon) {
+        if (this.ccInfoCloseBtn) {
+          this.ccInfoCloseBtn.blur();
+        }
+        this.ccInfoIcon.focus();
+      }
+    };
+
+    // eslint-disable-next-line prettier/prettier
+    this.setCCInfoCloseRef = (element) => {
+      this.ccInfoCloseBtn = element;
+    };
+    this.focusCCInfoCloseBtn = () => {
+      if (this.ccInfoCloseBtn) {
+        this.ccInfoIcon.blur();
+        this.ccInfoCloseBtn.focus();
+      }
+    };
+  }
+
+  componentDidUpdate() {
+    // Set focus on the correct element (508)
+    if (this.state.showBubble) {
+      this.focusCCInfoCloseBtn();
+    } else {
```

Comment:
I've been doing some testing locally, and think we can remove this `else` logic. It's causing the "i" button to be focused on first render, which isn't quite what we want. It should be re-focused when the close button is clicked.

Reply:
Yea, I was worried that might be the case. Was considering adding some other flag to the component to track where we were in terms of onLoad vs. having clicked the info button to try and prevent something like that.
I know we do need to track the focus setting in a lifecycle method as the way I manage whether or not to show the info bubble relies on React's internal re-rendering (by way of updating the `state` variable).

If removing this `else` fixes it? 🎉

Reply:
I had success removing the `else` block here and adding the `focusCCInfoIcon()` method in my comment below, so I think this should work as we want.

I like to try out code swaps locally before commenting, trying to offer good suggestions the first time instead of the third or fourth.